packages/sol-tracing-utils/src/utils.ts

```diff
@@ -85,34 +85,66 @@ export const utils = {
         if (_.isEmpty(structLogs)) {
             return structLogs;
         }
-        if (structLogs[0].depth === 1) {
-            // Geth uses 1-indexed depth counter whilst ganache starts from 0
-            const newStructLogs = _.map(structLogs, (structLog: StructLog, idx: number) => {
-                const newStructLog = {
+        const reduceDepthBy1 = (structLog: StructLog) => ({
+            ...structLog,
+            depth: structLog.depth - 1,
+        });
+        let normalizedStructLogs = structLogs;
+        // HACK(leo): Geth traces sometimes returns those gas costs incorrectly as very big numbers so we manually fix them.
```

Comment:
Is there an issue you can link to?

Reply:
No. They don't reply anyway ;) ask @albrow XD